app/controllers/organizations_controller.rb

```diff
@@ -109,4 +112,17 @@ def valid_filename?
     @organization.errors.add(:profile_image, FILENAME_TOO_LONG_MESSAGE)
     false
   end
+
+  def rate_limit!
+    rate_limiter.tap do |rate_limiter|
+      if rate_limiter.limit_by_action(:organization_creation)
+        retry_after = RateLimitChecker::RETRY_AFTER[:organization_creation]
+        raise RateLimitChecker::LimitReached, retry_after
+      end
+    end
+  end
+
+  def rate_limiter
+    RateLimitChecker.new(current_user)
```

Comment:
It's probably okay since we only call this from the context of one method, but if we end up using it elsewhere in the controller we might want to consider memoizing.